include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.cpp

```diff
@@ -732,13 +832,27 @@ SpatialOperatorBase<dim, Number>::get_constraint_p() const
   return constraint_p;
 }
 
+template<int dim, typename Number>
+dealii::AffineConstraints<Number> const &
+SpatialOperatorBase<dim, Number>::get_constraint_u() const
+{
+  return constraint_u;
+}
+
 template<int dim, typename Number>
 double
 SpatialOperatorBase<dim, Number>::get_viscosity() const
 {
   return param.viscosity;
 }
 
+template<int dim, typename Number>
+SpatialDiscretization const &
+SpatialOperatorBase<dim, Number>::get_spatial_discretization() const
+{
+  return param.spatial_discretization;
+}
+
```

Comment:
This function should be removed.

Reply:
But I will need it still in `postprocessor.cpp` since `Parameters param` is protected in `spatial_operator_base.h`, or am I missing something?

Reply:
ah ok. Then, I think it is best to implement a function `SpatialOperatorBase::distribute_constraints()` (or similar) so that the other two functions are not needed. This function is public and can either be used in the `TimeIntegrator` or the `PostProcessor`.

Reply:
Ah, missed this. I can move the two lines again from `PostProcessor` to `SpatialOperatorBase`. And then call the funtion from the same place.

Reply:
Here! https://github.com/exadg/exadg/pull/233/commits/dbfbaf9f1489c35dff9d9de8c34338011780e4ff#diff-b6700465d73837b3d857d6cb48936b37beff4706033534f21a6f4403039eb02c